mne/epochs.py

```diff
@@ -3212,7 +3217,7 @@ class Epochs(BaseEpochs):
     def __init__(
         self,
         raw,
-        events,
+        events=None,
```

Comment:
this change should be documented via `.. versionchanged::` and in the API changes section of the changelog

Reply:
You're right, @larsoner